src/accessor/tick.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = function() {
+  var date = function(d) { return d.date; },
+      high = function(d) { return d.high; },
+      low = function(d) { return d.low; },
+      spread = function(d) { return d.spread; };
```

Comment:
Yes, the spread can be derived.
I was thinking about hard coding the spread series into the actual tick chart much like the volume chart.
I was thinking about embedding the spread series in the tick plot because it's so important for many analysis that I am doing.